src/mygrad/_graph_tracking.py

```diff
@@ -0,0 +1,52 @@
+"""
+Provides user interface for suspending computational graph tracking and back-propagation
+"""
+from functools import wraps
+from typing import Callable
+
+import numpy as np
+
+__all__ = ["no_autodiff"]
+
+
+# If `False`, suspends all computational graph tracking and backprop
+TRACK_GRAPH = True  # type: bool
+
+
+class NoAutoDiff:
```

Comment:
```suggestion
class _NoAutoDiff:
```

Because I presume that this *really* shouldn't be touched from outside the module.